packages/sdk/src/types/tokens/ConnectorToken.ts

```diff
@@ -5,6 +5,7 @@ export interface ConnectorToken {
     content: unknown;
     createdAt: string;
     expiresAt: string;
+    forIdentity?: string;
```

Comment:
I guess strictly speaking this and the change in CreateTokenForOwnRelationshipTemplateRequest belong to #294 , but since this will hopefully be finished soon, too, I don't care too much.

Reply:
ehm, no. You can have a personalized token for a template so this has to be added now